src/pivotal-ui/components/header/header.scss

```diff
@@ -143,14 +143,19 @@ header ul li {
   padding: 0;
   margin: 0;
 }
+
 header a {
   color: $header-subtle-text-color;
+  &:focus {
+    outline: none;
```

Comment:
And removed. The reason for it being there was fixed by fixing another bug (the z-index bootstrap wtf)